src/test.cpp

```diff
@@ -207,6 +207,57 @@ Markdown lol \\|\\|spoiler\\|\\| \\~\\~strikethrough\\~\\~ \\`small \\*code\\* b
 		set_test("PERMISSION_CLASS", success);
 	}
 
+	{ // some dpp::user methods
+		dpp::user user1;
+		user1.id = 189759562910400512;
+		user1.discriminator = 0001;
+		user1.username = "brain";
+
+		set_test("USER.GET_MENTION", false);
+		set_test("USER.GET_MENTION", user1.get_mention() == "<@189759562910400512>");
+
+		set_test("USER.FORMAT_USERNAME", false);
+		set_test("USER.FORMAT_USERNAME", user1.format_username() == "brain#0001");
```

Comment:
this should really fail... usernames are case sensitive, it's possible to have both brain#0001 and Brain#0001 at the same time (I know because I met the lower case me!)